mappings/net/minecraft/client/gl/GlUniform.mapping

```diff
@@ -0,0 +1,34 @@
+CLASS none/bwp net/minecraft/client/gl/GlUniform
+	FIELD a LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD b loc I
+	FIELD c count I
+	FIELD d dataType I
+	FIELD e intData Ljava/nio/IntBuffer;
+	FIELD f floatData Ljava/nio/FloatBuffer;
+	FIELD g name Ljava/lang/String;
+	FIELD h stateIsDirty Z
+	FIELD i program Lnone/bwk;
+	METHOD <init> (Ljava/lang/String;IILnone/bwk;)V
+		ARG 0 name
+		ARG 1 dataType
+		ARG 2 count
+		ARG 3 program
+	METHOD a getName ()Ljava/lang/String;
+	METHOD a setFloat1 (F)V
```

Comment:
It's not a particularly thick layer though.

If you want to see what it's like in practice, have a look at none/bwm.

If we are taking the overloading route, I'd rather see `setFloats` rather than `setFloat`, and even then it kinda looks out of place for a 1-element vector.